pages/linux/sstat.md

```diff
@@ -0,0 +1,16 @@
+# sstat
+
+> View information about running jobs.
```

Comment:
Only for `running` jobs?
```suggestion
> View information about running jobs.
```

Reply:
Sorry, but I can't see your answer to this @npalladium

Reply:
As far as I know, it is for running jobs. The man pages say the same.